docs/contract-test-collection.md

```diff
@@ -188,73 +396,261 @@ storage (map
                                               (pair key_hash (pair timestamp address))))))))
           int);
 ```
-Note the lack of annotations. If the storage does not annotate its properties, the caller must use numeric indexes instead.
+
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+    const storageMap = new MichelsonMap();
+    storageMap.set(
+      {
+        0: '1',
+        1: '2',
+        2: 'test',
+        3: 'cafe',
+        4: '10',
+        5: true,
+        6: user_addresses.get('Eddy'),
+        7: '2019-09-06T15:08:29.000Z',
+        8: user_addresses.get('Eddy'),
+      },
+      100
+    );
+
+    storageMap.set(
+      {
+        0: '10',
+        1: '20',
+        2: 'Hello',
+        3: 'ffff',
+        4: '100',
+        5: false,
+        6: user_addresses.get('Freda'),
+        7: '2019-10-06T15:08:29.000Z',
+        8: user_addresses.get('Freda'),
+      },
+      1000
+    );
+
+    const op = await tezos.contract.originate({
+      code: contractMap8pairs,
+      storage: storageMap,
+    });
+  ```
+</TabItem>
+</Tabs>
+
+Note the lack of annotations in the Michelson for the storage. If the storage does not annotate its properties, the caller must use numeric indexes instead, as the Taquito javascript shows.
 
 ## MapWithInitialStorageContract
 
 [KT1Xjhpt7EB4kZvoSaeYE4zi47wYcdMAuDjD](https://better-call.dev/hangzhou2net/KT1Xjhpt7EB4kZvoSaeYE4zi47wYcdMAuDjD/code)
 
+Taquito provides a get method of the MichelsonMap on storage of type Map. We can only change contract storage by calling the function provided by the contract. The main function on this Smart Contract is decreasing the value of the current_stock associated with the key 1.
+
 #### Entrypoints:
-  * default
 
-#### Storage:
+- default
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'}
+]}>
+<TabItem value="michelson">
+
 ```js
-storage (map nat (pair (nat %current_stock) (mutez %max_price)));
+storage (pair (big_map %thebigmap (pair nat address) int)
```

Comment:
The storage doesn't correspond to `KT1Xjhpt7EB4kZvoSaeYE4zi47wYcdMAuDjD`. It should be: `(map nat (pair (nat %current_stock) (mutez %max_price)))`

Reply:
fixed.

Reply:
Similar comment for the Taquito Storage representation.
The following storage doesn't match
```
storage: {
    thebigmap: storageBigMap,
    themap: storageMap,
  }
```

Reply:
set the taquito for the example as
const storageMap = new MichelsonMap();
storageMap.set('1', { current_stock: '10000', max_price: '50' });
storageMap.set('2', { current_stock: '120', max_price: '20' });
storageMap.set('3', { current_stock: '50', max_price: '60' });